lib/lita/handlers/jarvis.rb

```diff
@@ -0,0 +1,21 @@
+require "jarvis/commands/merge"
+require "jarvis/commands/bounce"
+require "jarvis/mixins/fancy_route"
+
+module Lita
+  module Handlers
+    class Jarvis < Handler
+      extend ::Jarvis::Mixins::FancyRoute
+
+      on :loaded, :loaded
+      def loaded(*args)
+        WorkPool.setup_singleton
+      end
```

Comment:
I don't think we have concurrency issue, so I will try to rely on the Singleton module instead.